changelog.rst

```diff
@@ -1,5 +1,9 @@
 .. index::
     single: CHANGELOG
+    
+.. !! CAUTION !!
+   This file is automatically generated each month. Do not add new changelog
```

Comment:
I'd remove `each month` to not constraint ourselves and be able to change the frequency as needed.